### DISCUSSION TITLE
Wie kann man mit YCom ganze Medienpool-Ordner schützen anstatt nur einzelne Dateien?

### DISCUSSION THREAD

**Discussion by danspringer** [FriendsOfREDAXO/.github#26] [Q&A] [answered]
In der YCom-Doku ist der Dateischutz mit dem MediaAuth-Plugin beschrieben. Dies umfasst aber nur den Schutz einzelner, ausgewählter Dateien.
Dieser Dateischutz muss dann im Medienpool für jede Media-Datei einzeln gesetzt werden, was teilweise unpraktisch ist.

Wie kann man gesamte Ordner inkl. Unterordner mit YCom schützen?

**Comment by skerbis** [comment, marked as answer]
Hier eine Alternative: https://friendsofredaxo.github.io/tricks/addons/ycom/protected_files
MediaAuth-Plugin wird nicht benötigt
